recipe/meta.yaml

```diff
@@ -25,26 +28,29 @@ requirements:
     - python                                 # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
```

Comment:
We do not build cross-platform so we can remove python and cython from the build environment